.circleci/config.yml

```diff
@@ -5,6 +5,10 @@ parameters:
     type: enum
     enum: [release, publish_rpms, pull_requests]
     default: pull_requests
+  gio_product:
```

Comment:
⚠️ some of the config has been moved to https://github.com/gravitee-io/gravitee-access-management/blob/master/.circleci/workflows.yml

Make sure to rebase